pages/common/qmmp.md

```diff
@@ -0,0 +1,32 @@
+# qmmp
+
+> An audio player.
+> More information: <http://qmmp.ylsoftware.com>.
+
+- Launch qmmp:
```

Comment:
```suggestion
- Launch player:
```


- This change is optional - no corresponding style guide requirement exists.
- As the whole page describes `qmmp` there is no need to mention it in this description explicitly.